src/include/switch_utils.h

```diff
@@ -1346,12 +1346,12 @@ SWITCH_DECLARE(void *) switch_calloc(size_t nmemb, size_t size);
 
 /* malloc or DIE macros */
 #ifdef NDEBUG
-#define switch_malloc(ptr, len) (void)( (!!(ptr = malloc(len))) || (fprintf(stderr,"ABORT! Malloc failure at: %s:%d", __FILE__, __LINE__),abort(), 0), ptr )
-#define switch_zmalloc(ptr, len) (void)( (!!(ptr = calloc(1, (len)))) || (fprintf(stderr,"ABORT! Malloc failure at: %s:%d", __FILE__, __LINE__),abort(), 0), ptr)
+#define switch_malloc(ptr, len) (void)( ((ptr = malloc(len))) != NULL ? ptr : (fprintf(stderr,"ABORT! Malloc failure at: %s:%d", __FILE__, __LINE__), abort(), ptr))
```

Comment:
I don't see the benefit of changing this lines, it can make this readable but it would also be confusing soon when we try to merge from upstream.